opt/swagger-ui/dist/oauth2-redirect.html

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html lang="en-US">
+<head>
+    <title>Swagger UI: OAuth2 Redirect</title>
+</head>
+<body>
+<script>
```

Comment:
Suddenly made me think of [scittle](https://babashka.org/scittle/) (not recommending you change it of course!).

Reply:
Cool, never heard of scittle!